### PR TITLE
OS X fix

### PR DESCRIPTION
**Pull request by bjones1**
Fix: Corrections on OS X courtesy of @yanamal. See #1944.

### CODE REVIEW

**Comment by bnmnetp** [comment]
Is this ready?

**Comment by yanamal** [comment]
I just tested it by resetting my group membership (hopefully) back to the way it was before Runestone and re-running the script.

It works, with one caveat: because of sudo, it stops everything and asks for my password in the terminal when it reaches that point. Not sure if this is bad for users who might not expect it? (e.g. they start the script and then switch tabs). I think it does need sudo privileges to do the commands, but I'm not 100% sure of that either.

**Comment by bjones1** [comment]
@yanamal, thanks for testing! That's typical for this script -- it will need sudo permission in most cases, and will (hopefully) ask for it almost immediately after starting, so that users won't have switched away from that window.

**Comment by bjones1** [comment]
@bnmnetp, do you want to review this before merge, given your a Mac person?

File: docker/docker_tools.py [hunk 1]
```diff
@@ -87,8 +92,8 @@ def check_install(
         print("Not found. Installing...")
         subprocess.run(
             # Only run with ``sudo`` if we're not root.
-            ([] if is_root else ["sudo"]) +
-            [
+            ([] if is_root else ["sudo"])
+            + [
                 "apt-get",
                 "install",
                 "-y",
```
Comment (bnmnetp) [inline docker/docker_tools.py:99]: I think this is missing the install_package

**Pull request merged**